packages/mdc-slider/foundation.js

```diff
@@ -244,10 +268,16 @@ export default class MDCSliderFoundation extends MDCFoundation {
     }
   }
 
+  /**
+   * Creates a handler for mouse/touch/pointer down events
+   * @param {string} moveEvt
+   * @param {(function(!Event): !number)=} getPageX
```

Comment:
Primitive values like `number` are non-null by default, so you can remove the `!` qualifier (ditto below)